### PR TITLE
Notify bone coordinate

### PR DESCRIPTION
**Pull request by Rui914**
#403
Add show_bone_pos = (true/false)

### CODE REVIEW

File: minetest.conf.example [hunk 1]
```diff
@@ -26,3 +26,6 @@
 # Enable the stairs mod ABM that replaces the old 'upside down'
 # stair and slab nodes in old maps with the new param2 versions.
 #enable_stairs_replace_abm = false
+
+#Notify player with the location of their dropped bones
```
Comment (sfan5) [inline minetest.conf.example]: Minor thing: It's usually done
```py
# Like this (note the extra space between # and the text)
```

**Comment by sfan5** [comment]
:+1:

**Comment by PilzAdam** [comment]
:-1:, see https://github.com/minetest/minetest_game/pull/403#issuecomment-74158976

**Comment by C1ffisme** [comment]
Yes @PilzAdam , but it's customizable. If you don't like it, you don't put it on your server.

**Comment by PilzAdam** [comment]
@C1ffisme game design shouldn't be done by server admins.

**Comment by C1ffisme** [comment]
@PilzAdam What do you think minigame servers do? Do they still have the same building, crafting and mining setup?

Short Awnser : No.

The truth is, if you own a copy of a game, you can play it however you want. If you want to use TNT to destroy everything or build buildings that look almost non-voxel, you can do that. Besides, you can already choose your game's design with texture packs and mods. You could even rewrite the entire game differently if you wanted to. You can't say that game design isn't done by server admins, because the server admins are the ones who can code the game they are hosting.

If you say game design shouldn't be done by server admins or players, that's basically saying we shouldn't be able to mod or use a different subgame.

**Comment by FreeLikeGNU** [comment]
:+1:  Knowing where bones drop is A Good Thing™ for the servers I run.

**Comment by PilzAdam** [comment]
@C1ffisme minigames are not minetest_game. If you install mods, then the mod creator takes care that it works good (game design wise) with the game it modifies. If you go ahead and modify the game yourself, then you become a game designer; providing a setting however implies that all users, not just the ones that are comfortable in game design, can and will modify the game.

**Comment by C1ffisme** [comment]
Yes. It can modify the game and some people might think that others aren't game designers. But many good games have things that let you get overpowered (Minecrafts 'keepInventory' rule, Age of Empires "Post Iron Age", etc.). If someone decides they want information on where they died, then that is their choice. Whether chat is their style or not is irrelevant. We can gussy up the design of showing bones later, but for now I think people deserve to use features as they wish.

Making a game is about balancing it, but letting people customize it is about giving a much control as possible.

File: mods/bones/init.lua [hunk 2]
```diff
@@ -181,6 +181,12 @@ minetest.register_on_dieplayer(function(player)
 	end
 	
 	minetest.set_node(pos, {name="bones:bones", param2=param2})
+
+	if minetest.setting_getbool("show_bone_pos") == true then
```
Comment (rubenwardy) [inline mods/bones/init.lua]: ommit `== true`, it's superfluous.

**Comment by Ferk** [comment]
It would be really cool if we could have a (hide-able) overlay of tiny icons in the minimap with an API to add them.

Then we could have an icon to show the position of the bones in x,z  as well as the sethome place, and maybe other things that other mods could add.

I don't think giving the coordinates is very useful, because the users don't actually know their current coordinates in the first place. Unless they enable show_debug, which is not a good thing to ask from them.

**Comment by Rui914** [comment]
This feature can be implemented by the mod.

**Comment by FreeLikeGNU** [comment]
Yay!

**Pull request closed without merging**